tests/ui/attributes/mixed_export_name_and_no_mangle.stderr

```diff
@@ -0,0 +1,19 @@
+warning: the `#[export_name]` attribute may not be used in combination with `#[no_mangle]`
+  --> $DIR/mixed_export_name_and_no_mangle.rs:6:1
+   |
+LL | #[export_name = "foo"]
+   | ^^^^^^^^^^^^^^^^^^^^^^ `#[export_name]` takes precedence
+   |
+note: `#[no_mangle]` is ignored
+  --> $DIR/mixed_export_name_and_no_mangle.rs:5:1
+   |
+LL | #[no_mangle]
+   | ^^^^^^^^^^^^
+   = note: `#[warn(mixed_export_name_and_no_mangle)]` on by default
```

Comment:
```suggestion
   = note: `#[warn(unused_attributes)]` on by default
```

Reply:
With the `unused_attributes` now in use, would it also make sense to switch the "unused" attribute to the primary span and make the `export_name` attribute then the secondary? So in fact just switch the 2 attributes in the error message.

My idea was also that it would make most sense that the  `#[allow(unused_attributes)]` then is put to the `#[no_mangle]` attribute and not to the `#[export_name]` attribute.

Would you agree on that?

Reply:
Yeah, that makes sense. Let's put the warning on the `#[no_mangle]`.

Reply:
implemented with 338f8f73da8b39d19ce4f3e527a076ef79857005